File: Luyencode/DK08 - Máy tính bỏ túi đơn giản/DK08.cs
using System;

namespace PTB2
{
    public class Program
    {
        static void Main(string[] args)
        {
            // Đọc và xử lý chuỗi đầu vào
            string[] input = Console.ReadLine().Split(' ');
            // Lấy các hệ số a, b, c từ mảng
            double a = double.Parse(input[0]);
            double b = double.Parse(input[2]);
            if(input[1] == "+")
                Console.Write("{0:0.00}", a + b);
            else if(input[1] == "-")
                Console.Write("{0:0.00}", a - b);
            else if(input[1] == "*")
                Console.Write("{0:0.00}", a * b);
            else if(input[1] == "/")
            {
                if(b == 0)
                    Console.Write("Math Error");
                else
                    Console.Write("{0:0.00}", a / b); 
            }      
        }
    }
}

File: Luyencode/DK08 - Máy tính bỏ túi đơn giản/DK08.py
# Đọc và xử lý chuỗi đầu vào
input_data = input().split(' ')

# Lấy các giá trị a, b và toán tử
a = float(input_data[0])
operator = input_data[1]
b = float(input_data[2])

# Xử lý toán tử và in kết quả
if operator == "+":
    print(f"{a + b:.2f}")
elif operator == "-":
    print(f"{a - b:.2f}")
elif operator == "*":
    print(f"{a * b:.2f}")
elif operator == "/":
    if b == 0:
        print("Math Error")
    else:
        print(f"{a / b:.2f}")
